app/code/Magento/Sales/etc/adminhtml/system.xml

```diff
@@ -48,9 +48,9 @@
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
             </group>
-            <group id="zerograndtotal_creditmemo" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
+            <group id="allow_zero_grandtotal" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Unfortunately, we are not allowed to remove/rename configuration according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/). Any other 3d party extension that depends on `sales/zerograndtotal_creditmemo/allow_zero_grandtotal` config will be broken.